docs/custom-domains.md

```diff
@@ -1,26 +1,64 @@
 ---
 id: custom-domains
-title: Set a Custom Domain
-sidebar_label: Set a Custom Domain
+title: Use a Custom Domain
+sidebar_label: Use a Custom Domain
 ---
 
-If you'd like to use a custom domain instead of `<your_canvas>.builtwithdark.com`, you can!
+Your backend endpoints are always available at `<your_canvas_name>.builtwithdark.com/*`.
 
-- We'll manage SSL for you - no need to buy your own cert.
-- You need to set up a CNAME from your desired domain to `<your_canvas>.darkcustomdomain.com`.
+If you'd like, you can additionally use a custom domain like `www.myawesomedarkservice.com` or `api.myawesomedarkservice.com` to serve your endpoints!
 
-  - Note: this cannot be an apex (`foo.com`); using `www.foo.com` is the usual
-    way, though you could use `app` or `api` or another subdomain instead of
-    `www`. Why? Netlify has a post about this
-    ([https://www.netlify.com/blog/2017/02/28/to-www-or-not-www/](https://www.netlify.com/blog/2017/02/28/to-www-or-not-www/)), but tl;dr apex
-    CNAMEs aren't supported by the DNS spec, and A records remove some of our
-    options for providing stable and resilient infrastructure.
+We'll manage SSL for you - no need to buy your own certificate to make your site or service available over `https`.
 
-- You can, if you wish, also set up an A record pointing the apex (`foo.com`) to
-  `35.227.208.117`; we cannot currently provide an SSL cert for that, but we can
-  redirect to your main subdomain (usually `www`). If your DNS provider supports
-  ALIAS records (like CNAME, but permitted on an apex), that also works - though
-  again, for redirects, not SSL certs.
+## How to set it up
 
-- Once DNS resolves, [contact us](support) with your domain and canvas, and
-  we'll set things up on our end.
+1. Set up a `CNAME` record that points from your desired domain to `<your_canvas_name>.darkcustomdomain.com`.
+   For example, if your canvas is called `ellen-demo`, point your `CNAME` at `ellen-demo.darkcustomdomain.com`.
+   The specifics for this step depends on how you manage your DNS settings, but here's what it might look like on Cloudflare:
+
+   ![Setting a CNAME on Cloudflare](assets/customdomain/cname-example.png)
+
+   **Note**: You _cannot_ use an apex domain like `myawesomedarkservice.com`. Using `www.myawesomedarkservice.com` is the usual
+   way, although you can use `app`, `api`, or another subdomain instead of `www`.
+   This is because apex `CNAME`s aren't supported by the DNS spec, and `A` records remove some of our
+   options for providing stable and resilient infrastructure.
+   You can read [this Netlify post](https://www.netlify.com/blog/2017/02/28/to-www-or-not-www/) for more information.
+
+   If you wish, you can additionally set up an `A` record pointing the apex (eg `myawesomedarkservice.com`) to
+   `35.227.208.117`. We cannot currently provide an SSL cert for that, but we can
```

Comment:
Is this still the correct IP? When using dig on a correctly configured domain I got `35.197.124.122` instead of `35.227.208.117`